ftdetect/bzl.vim

```diff
@@ -12,4 +12,7 @@
 " See the License for the specific language governing permissions and
 " limitations under the License.
 
-autocmd BufRead,BufNewFile *.bzl,BUILD,*.BUILD,BUILD.*,WORKSPACE setfiletype bzl
+autocmd BufRead,BufNewFile *.bzl,WORKSPACE setfiletype bzl
+if has('fname_case')
+  autocmd BufRead,BufNewFile BUILD,*.BUILD,BUILD.* setfiletype bzl
```

Comment:
OSX doesn't have fname_case? I didn't see it mentioned in vim's help: "for Amiga, MS-DOS, and Windows this is not present".